.circleci/build-all-commits.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Abort on error, and error if variable is unset
+set -eu
+
+. ./helpers.sh
+
+logfile=$(createLog circleci build)
+branch=$(git rev-parse --abbrev-ref HEAD)
+# Skip the latest commit as it will be automatically built by the branch being pushed.
+mapfile -t commits < <(git log "develop...HEAD~1" --format="%h")
+
+for commit in "${commits[@]}"
+do
+    curl \
+        -u "${CIRCLECI_API_USER_TOKEN}:" \
+        --data "revision=$commit" \
+        --data "branch=$branch" \
+        "https://circleci.com/api/v1.1/project/github/seriema/retro-cloud/build" \
+    | tee -a "$logfile"
+done
```

Comment:
Here is an example of what I see the use of this. After some rebasing and cleaning the final changes in a branch might pass all tests, but individual commits are broken. If I'd like to revert or cherry-pick, or just study, I'd like them to be complete set of works. Being able to re-run them with this script (`build-all-commits.sh`) shows me which ones I broke.
![image](https://user-images.githubusercontent.com/693684/79686114-35c66880-823e-11ea-8e50-ad0129b37709.png)